### DISCUSSION TITLE
Is there any way to disable fullscreen button ? in MediaElement

### DISCUSSION THREAD

**Discussion by NykuluzM** [CommunityToolkit/Maui#1852] [General]
In my use case i just want quick preview and i put the media player in a popup it messes up with the popup if the full screen button is clicked it would help me if someone can help me remove it

**Comment by CliffAgius** [comment]
How about disable the controls and then adding your own so you can control the buttons the user has access to.

Take a look at the docs here:

https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/mediaelement?tabs=windows#implement-custom-transport-controls

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
